addBag.php

```diff
@@ -19,26 +25,40 @@
         <form method ="post">
             <div>
                 <label>Brand: </label>
-                <input type="text" name="brand" required/><br>
+                <input type="text" name="brand" /><br>
             </div>
             <div>
                 <label>Model: </label>
-                <input type="text" name ="model" required/><br>
+                <input type="text" name ="model" /><br>
             </div>
             <div>
                 <label>Price (£): </label>
-                <input type="text" name ="price" required/><br>
+                <input type="text" name ="price" /><br>
```

Comment:
You have a function which checks that none of these fields are empty, right? Are you sure that you don't want them to be required?

Reply:
i do! made validating hard so removed with the intention to add them back in! thanks for spotting!